docs: Improve suggestions about WSL2 docker options (#5952)
ddev/ddev: docs/content/users/install/ddev-installation.md

    You can install DDEV on Windows three ways:

    1. **Most reliable, free, open-source:** [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
    2. **Commercial, often problematic:** [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)

>     3. **Old school:** [Installing directly on traditional Windows](#traditional-windows) with an installer

Reviewer: ```suggestion
    3. **Old school:** [Installing without WSL2 on traditional Windows](#traditional-windows) with an installer
 ```

Author: I wonder how to say this better.  This goes back to the same confusion about WSL2. You can't install Docker Desktop (required for old-school Windows setup) without WSL2, because that's what it uses under the hood. But with this solution, **DDEV** is used from git-bash or PowerShell, and it uses Docker Desktop.

>     1. **Most reliable, free, open-source:** [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
>     2. **Commercial, often problematic:** [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
>     3. **Old school:** [Installing directly on traditional Windows](#traditional-windows) with an installer

Reviewer 2: ```suggestion
    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script) **Recommended, best performance**
    2. [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script) **May require licence **
    3. [Installing directly on traditional Windows](#traditional-windows) with an installer **Legacy, slower performance**
```

Author: ```suggestion
    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script) **Recommended, best performance, most reliable**
    2. [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script) **May require license, less reliable**
    3. [Installing directly on traditional Windows](#traditional-windows) with an installer **Legacy, slower performance**```

    You can install DDEV on Windows three ways:

    1. **Most reliable, free, open-source:** [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
    2. **Commercial, often problematic:** [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
    3. **Old school:** [Installing directly on traditional Windows](#traditional-windows) with an installer

>     **We strongly recommend using WSL2.** While its Linux experience may be new for some Windows users, it’s worth the performance benefit and common experience of working with Ubuntu and Bash.

Reviewer: **We strongly recommend using WSL2.**

I now get that it means as compared to WSL1. How about

```suggestion
    **We strongly recommend updating your WSL to WSL2 for the best performance.**

Author: Thanks for reviewing! No, actually WSL1 is not usable with DDEV or Docker, and none of the instructions will work with WSL1, and no docker platform will work there. 

```suggestion
    **We strongly recommend using WSL2 for your Windows DDEV development environment.** While its Linux experience may be new for some Windows users, it’s worth the performance benefit and common experience of working with Ubuntu and Bash.
```

    ### WSL2 + Docker Desktop Install Script

>     WSL2 with Docker Desktop is a less-favored choice because Docker Desktop is often buggy and lightly supported, and has many features not required for use with DDEV. It is also not free software (although smaller organizations can use it free of charge) and it is not open-source.

Author: ```suggestion
    WSL2 with Docker Desktop is a less-favored choice because Docker Desktop may be lightly supported, and has many features not required for use with DDEV that do not add particular value. It is also not free software (although smaller organizations can use it free of charge) and it is not open-source.
```
